### DISCUSSION TITLE
Option to not release alsa after playback

### DISCUSSION THREAD

**Discussion by ghost** [librespot-org/librespot#619] [Feature Requests]
Apologies if this is already possible, but I could not find any mention of this.

The DAC I'm using seems to go to sleep when ALSA is released and takes a few second to respond when the playback starts again. Is it possible to not release ALSA even when no music is playing?

Thanks :)

**Comment by Malvineous** [comment]
You'd be better off disabling this feature in ALSA as it affects all programs, not just librespot.  You can also investigate disabling power saving features on the sound card which can cause a similar delay.  There is lots of detail about this online as it's a common problem.

**Comment by giggywithit** [comment]
```
sudo nano /etc/modprobe.d/pmdowntime.conf
#30 min idle timeout - time in milliseconds
options snd_soc_core pmdown_time=1800000
```
